fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/domain/ScheduledJobDetailRepository.java

```diff
@@ -45,4 +45,7 @@
     @Query("select jobDetail from ScheduledJobDetail jobDetail where jobDetail.nodeId = :nodeId or jobDetail.nodeId = 0")
     List<ScheduledJobDetail> findAllJobs(@Param("nodeId") Integer nodeId);
 
+    @Query("select jobDetail from ScheduledJobDetail jobDetail where jobDetail.jobName = :jobName")
```

Comment:
No need for the custom Query. The generated Spring Data JPA one will be the same.